merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/model/ModelType.java

```diff
@@ -48,6 +50,10 @@ public interface ModelType<Config, Model> {
    */
   InputType<Config> getConfigurationType();
 
+  default List<String> getSubsystems() {
+    return Collections.emptyList();
+  }
```

Comment:
@mattdailis I'll defer to you if you think otherwise, but on Aerie we don't do `default` methods for interfaces. Please remove the "default" implementation:

```suggestion
  List<String> getSubsystems()
```

Reply:
Additionally, please add a doc comment to this method